src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -111,7 +113,21 @@ public void clickConfermaButtonEliminaPopUp() {
     public void waitLoadPopUp() {
         try {
             By titleBy = By.id("dialog-title");
+            By descriptionBy = By.id("dialog-description");
+            By titleOption = By.xpath("//div[@data-testid='dialog-content']//p[contains(text(), 'Inserisci codice')]");
```

Comment:
There is no `data-testid` for the `p` web element, because it is inside the content of the modal. If you only take `data-testid='dialog-content'`, it will fail because you're retrieving the entire `div` with all the elements inside.